tests/unit_tests/test_subtensor.py

```diff
@@ -766,44 +733,54 @@ def test_get_hotkey_owner_does_not_exist(mocker, subtensor):
     mocker.patch.object(subtensor, "does_hotkey_exist", return_value=False)
 
     # Call
-    result = subtensor.get_hotkey_owner(hotkey_ss58, block)
+    result = await subtensor.get_hotkey_owner(hotkey_ss58, block)
 
     # Assertions
     assert result is None
     subtensor.query_subtensor.assert_called_once_with("Owner", block, [hotkey_ss58])
     subtensor.does_hotkey_exist.assert_not_called()
 
 
-def test_get_hotkey_owner_no_block(mocker, subtensor):
+@pytest.mark.asyncio
+async def test_get_hotkey_owner_no_block(mocker, subtensor):
     """Test get_hotkey_owner with no block specified."""
     # Prep
     hotkey_ss58 = "test_hotkey"
     expected_owner = "coldkey_owner"
     mock_result = mocker.MagicMock(value=expected_owner)
-    mocker.patch.object(subtensor, "query_subtensor", return_value=mock_result)
-    mocker.patch.object(subtensor, "does_hotkey_exist", return_value=True)
+    mocker.patch.object(
+        subtensor, "query_subtensor", new=mocker.AsyncMock(return_value=mock_result)
+    )
+    mocker.patch.object(
+        subtensor, "does_hotkey_exist", new=mocker.AsyncMock(return_value=True)
+    )
```

Comment:
Shouldn't these be patched in a context manager?

Reply:
I haven't found a reason to limit the patch within a portion of the test. Regular use without a context manager works for us across of entire test module.